parsl/executors/status_handling.py

```diff
@@ -163,12 +161,8 @@ def handle_errors(self, status: Dict[str, JobStatus]) -> None:
         """
         if not self.block_error_handler:
             return
-        init_blocks = 3
-        if hasattr(self.provider, 'init_blocks'):
```

Comment:
removing this changes the behaviour of the simple error handler, right? so that it will always wait for the first three blocks, rather than the init_blocks number? If we care about this PR adding in new stuff, not changing old behaviour, then this test could move into simple_error_handler, because it has access to executor.provider.init_blocks there.

Reply:
Fixed.